lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -3115,6 +3117,179 @@ ParseResult FIRStmtParser::parseStaticRefExp(Value &result,
   return failure(parseIdOrInstance() ||
                  parseOptionalExpPostscript(result, false));
 }
+/// static_reference ::= id
+///                  ::= static_reference '.' id
+///                  ::= static_reference '[' int ']'
+/// Populate `refResult` with rwprobe "root" and parsed indexing.
+/// Root is base-type target, and will be block argument or forceable.
+/// Also set `Type`, so we can handle const-ness while visiting.
+/// If root is an unbundled entry, replace with bounce wire and update
+/// the unbundled entry to point to this for future users.
+// NOLINTNEXTLINE(misc-no-recursion)
+ParseResult FIRStmtParser::parseRWProbeStaticRefExp(FieldRef &refResult,
+                                                    Type &type,
+                                                    const Twine &message) {
+  auto loc = getToken().getLoc();
+
+  StringRef id;
+  SymbolValueEntry symtabEntry;
+  if (parseId(id, message) ||
+      moduleContext.lookupSymbolEntry(symtabEntry, id, loc))
+    return failure();
+
+  // Three kinds of rwprobe targets:
+  // 1. Instance result.  Replace with a forceable wire, handle as (2).
+  // 2. Forceable declaration.
```

Comment:
Presently all declarations are eagerly created "forceable" and demoted at end of parsing for the bulk that are not used in that way (rwprobe unused).

For now, continue this but hopefully soon can drop forceable altogether.

Just to clarify, (2) still puts symbol on it in the common case -- this just creates the bounce wire in the same way we would normally create a wire.